core/templates/pages/exploration-player-page/learner-experience/tutor-card.component.html

```diff
@@ -12,14 +12,10 @@
        [ngClass]="{'learner-view-card-transition-class': isOnTerminalCard(), 'card-content-hidden': !checkMarkHidden}">
     <div class="oppia-learner-view-card-top-section">
       <oppia-content-language-selector></oppia-content-language-selector>
-      <a [ngClass]="{'oppia-disabled-image-link': (OPPIA_AVATAR_LINK_URL === null)}"
-         [href]="OPPIA_AVATAR_LINK_URL"
-         target="_blank"
-         rel="noopener">
-        <img class="conversation-skin-oppia-avatar"
-             [src]="OPPIA_AVATAR_IMAGE_URL" alt=""
+      <img class="conversation-skin-oppia-avatar"
+           [src]="OPPIA_AVATAR_IMAGE_URL"
+           alt="">
              *ngIf="avatarImageIsShown">
```

Comment:
Where does this belong to now? This needs to be part of some tag, right?

Reply:
Ah! sorry for that @vojtechjelinek the '>' after ' alt="" ' was by mistake putted by me I have removed it now.